src/content/docs/extensions/iceberg.mdx

```diff
@@ -0,0 +1,122 @@
+│ version_name_format           │ STRING    │ 'v%s%s.metadata.json,%s%s.metadata.json' │ Controls how versions are converted to metadata file names │
+└───────────────────────────────┴───────────┴──────────────┴──────────────────────────────────────────────────────────────────────────────┘
+```
+
+### Scan the Iceberg file
+
+```cypher
+LOAD FROM 'kuzu/extension/iceberg/test/iceberg_tables/person_table' (file_format='iceberg', allow_moved_paths=true) RETURN *;
+```
+```
+┌─────────┬───────┬──────────────┬─────────────────────────────┐
+│ Name    │ Age   │ Balance      │ Date of Birth               │
+├─────────┼───────┼──────────────┼─────────────────────────────┤
+│ Alice   │ 25    │ 1000.500000  │ 1999-01-15 05:00:00+00      │
+│ Bob     │ 30    │ 1500.750000  │ 1994-06-25 04:00:00+00      │
+│ Charlie │ 35    │ 2000.000000  │ 1988-12-05 05:00:00+00      │
+│ Diana   │ 40    │ 2500.300000  │ 1983-07-20 04:00:00+00      │
+│ Ethan   │ 28    │ 1800.600000  │ 1996-03-10 05:00:00+00      │
+└─────────┴───────┴──────────────┴─────────────────────────────┘
+
+```
+
+:::note[Note]
+The `allow_moved_paths` option ensures that some path resolution is performed, which allows scanning Iceberg tables that are moved.
+:::
+
+This extension can be paired with the httpfs extension to access Iceberg tables in object stores such as S3.
+
+```
+LOAD FROM 's3://kuzu-test/iceberg' (file_format='iceberg', allow_moved_paths=true) RETURN *;
+```
+
+### Access Iceberg Metadata
+
+```cypher
+CALL ICEBERG_METADATA('kuzu/extension/iceberg/test/iceberg_tables/lineitem_iceberg', allow_moved_paths := true) return *;
+```
+
+```
+┌──────────────────────────┬──────────────────────────┬──────────────────┬─────────┬──────────┬──────────────────────────┬─────────────┬──────────────┐
+│ manifest_path            │ manifest_sequence_number │ manifest_content │ status  │ content  │ file_path                │ file_format │ record_count │
+│ STRING                   │ INT64                    │ STRING           │ STRING  │ STRING   │ STRING                   │ STRING      │ INT64        │
+├──────────────────────────┼──────────────────────────┼──────────────────┼─────────┼──────────┼──────────────────────────┼─────────────┼──────────────┤
+│ lineitem_iceberg/meta... │ 2                        │ DATA             │ ADDED   │ EXISTING │ lineitem_iceberg/data... │ PARQUET     │ 51793        │
+│ lineitem_iceberg/meta... │ 2                        │ DATA             │ DELETED │ EXISTING │ lineitem_iceberg/data... │ PARQUET     │ 60175        │
+└──────────────────────────┴──────────────────────────┴──────────────────┴─────────┴──────────┴──────────────────────────┴─────────────┴──────────────┘
+
+```
+
+:::note[Note]
+Be sure to use `:=` operator to set the variable in `CALL` function, as `=` operator in `CALL` function has a different meaning of comparison.
```

Comment:
Did we document this somewhere else along with CALL? I guess this is a common grammar rule not specific to iceberg but all CALL functions. @acquamarin Can you double check?

Reply:
keep this as non resolved as a reminder

Reply:
I'll keep a track of this separately.